lib/ethpy/ethpy/hyperdrive/interface/read_interface.py

```diff
@@ -320,6 +320,29 @@ def get_vault_shares(self, block_number: BlockNumber | None) -> FixedPoint:
             block_number = self.get_block_number(self.get_current_block())
         return _get_vault_shares(self.yield_contract, self.hyperdrive_contract, block_number)
 
+    def get_idle_shares(self, block_number: BlockNumber | None) -> FixedPoint:
```

Comment:
You'll want to double check with @dpaiton here, but I think the read interface might be strictly contract calls without math on our end

Reply:
We don't have a rust function for this yet, so I think it's ok to do it in python. However it's not using an RPC to get the balance so I think the docstring is incorrect.

Reply:
Added an issue to hyperdrive-sdk: https://github.com/delvtech/hyperdrive-sdk/issues/55